habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -155,13 +155,74 @@ class TeleportActionConfig(ActionConfig):
 @attr.s(auto_attribs=True, slots=True)
 class VelocityControlActionConfig(ActionConfig):
     type: str = "VelocityAction"
-    # meters/sec
-    lin_vel_range: List[float] = [0.0, 0.25]
-    # deg/sec
-    ang_vel_range: List[float] = [-10.0, 10.0]
+    lin_vel_range: List[float] = [0.0, 0.25]  # meters/sec
+    ang_vel_range: List[float] = [-10.0, 10.0]  # deg/sec
+    time_step: float = 0.1  # seconds
+    enable_scale_convert: bool = False
+    # Stop criteria
+    use_stop_heuristic: bool = False
     min_abs_lin_speed: float = 0.025  # meters/sec
-    min_abs_ang_speed: float = 1.0  # # deg/sec
-    time_step: float = 1.0  # seconds
+    min_abs_ang_speed: float = 1.0  # deg/sec
+
+
+@attr.s(auto_attribs=True, slots=True)
+class WaypointControlActionConfig(VelocityControlActionConfig):
+    r"""
+    In Navigation tasks only, this action will utilize a velocity controller to
+    move the robot to the desired waypoint.
+    The waypoint is specified in the frame of the robot.
+    """
+    type: str = "WaypointAction"
+    action_duration: float = 1.0  # seconds
+    # Action space range
+    waypoint_lin_range: List[float] = [-0.5, 0.5]  # meters
+    waypoint_ang_range: List[float] = [-180, 180]  # degrees
+    max_max_duration: float = 10.0  # seconds
```

Comment:
Consider changing the name of this variable to be more informative?

Reply:
Any suggestions?